### DISCUSSION TITLE
Tracking specific URL tag

### DISCUSSION THREAD

**Discussion by KrisLowet** [plausible/analytics#1546] [Anything Else]
Hi

My affiliate program uses the ?via= tag in the URL. How can I track this tag, just like the "ref" tag?

Thanks

**Comment by metmarkosaric** [comment]
hi @KrisLowet! that's not possible unfortunately. for privacy purposes, we automatically remove all the parameters except those that we support. ref, source, utm_source, utm_medium, utm_campaign query parameters are all valid and supported (utm_term and utm_content will be supported in a day or two as well). we might expand this list in the future but there are no short term plans that i can tell you about. see this thread https://github.com/plausible/analytics/discussions/177

**Reply by KrisLowet** [reply]
Okay, thanks for the info!
